### ISSUE TITLE
Add columen with language information

### ISSUE THREAD

**Issue by apohllo** [eugeneyan/open-llms#43]
It would be very valuable, if the table included information about language of the model/dataset.
While obviously many people are interested in English, some of the models support/were trained on languages different than English.

**Comment by LudwigStumpp** [comment]
I get your point but I think that specifying all languages that the model is trained on is infeasible as:
- table would be a mess
- large text corpara can have a ton of different languages in it, without even realizing it

**Comment by SamikshyaKarki0** [comment]
Yeah I agreee with you as well. It would be mess if you add more in there.

**Comment by apohllo** [comment]
Hmm, I understand you point, but in that case, we could just put "multilingual" as the value for the column, without specifying all the languages. And usually there is a group of languages that dominates the dataset (say 90% is English or Python). Having that information still would be beneficial I suppose.

**Comment by SamikshyaKarki0** [comment]
Ok, thanks for the clarification.
